pyscf/tdscf/uhf.py

```diff
@@ -782,6 +1057,181 @@ def vind(xys):
 
     return vind, hdiag
 
+@lib.with_doc(rhf.TDA.__doc__)
+class TDA_SF(TDBase):
+    singlet = None
+    
+    def gen_vind(self, mf=None,extype=None):
+        '''Generate function to compute Ax'''
+        if mf is None:
+            mf = self._scf
+        if extype is None:
+            extype = self.extype
+        return gen_tda_hop_sf(mf, wfnsym=self.wfnsym,extype=self.extype)
+
+    def init_guess0(self, mf, nstates=None, wfnsym=None,extype=None):
```

Comment:
Please add a docstring for extype

Reply:
I add a description above 'extype = self.extype' like this 
'extype is used to determine which spin flip excitation will be calculated.'
'spin flip up: exytpe=0, spin flip down: exytpe=1.'